ui/00_message/ui/clan_edit.xml

```diff
@@ -81,6 +89,9 @@
     <original>注意事項
 ・半角２文字～１２文字まで入力可能です
 ・半角英数字と一部の記号 '.,:;!?&amp;-_ が使用可能です</original>
+    <translation>Notes:
+・You can input up to 2 to 12 characters.
+・Half-size alphanumeric characters and some symbols '.,:;!?&amp;-_ can be used.</translation>
```

Comment:
It's weird, as the thing does say half-width, but you can use uppercase ones as well as lowercase ones in the naming, so it's strange what it's trying to imply here. Perhaps I should omit the "half-size thing here since most people don't use Japanese characters in their name" up to you on this.